index.html

```diff
@@ -111,13 +111,14 @@ <h1>Firmware</h1>
           <tr>
             <th>Hersteller</th>
             <th>Modell</th>
+            <th>Bootloader-Image</th>
```

Comment:
Well, the `Image` suffix feels inconsistent. But on the other hand we do not offer Bootloaders to download. So i think this is fine.

Reply:
I used the same naming as in the Wizard.

https://github.com/freifunk-darmstadt/gluon-firmware-selector/blob/master/app.js#L120